app/styles/app.css

```diff
@@ -248,17 +248,38 @@ input {
 
 .searchableMetricSelector .searchBar {
     border-bottom: 1px solid #eee;
+    position: sticky;
+    top: 0;
+    background-color: white;
+}
+
+.searchableMetricSelector .metricGroup {
+    font-weight: bold;
 }
 
-.searchableMetricSelector .searchResults p {
+.searchableMetricSelector .metricResult {
     border-bottom: 1px solid #D3D3D3;
 }
 
-.searchableMetricSelector .searchResults p:hover {
+.searchableMetricSelector .metricResult:hover{
+    background-color: rgba(180, 180, 180, 0.5);
+    cursor: pointer;
+}
+.searchableMetricSelector .metricChoice:hover{
     background-color: rgba(180, 180, 180, 0.5);
     cursor: pointer;
 }
 
+.metricResult .metricMetaData p {
+    margin-top: 2px;
+    margin-bottom: 2px;
+}
+
+.searchResults .metricMetaData {
+    font-size: 0.8em;
+    padding-left: 20px;
```

Comment:
For this and the above, try to use units of `em` instead of `px` so that the design scales correctly if the user changes their default browser font size.